hs_core/migrations/0065_auto_20221004_1707.py

```diff
@@ -22,12 +22,12 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='contributor',
             name='is_active_user',
-            field=models.BooleanField(default=False),
+            field=models.NullBooleanField(default=False, null=True),
         ),
         migrations.AddField(
             model_name='creator',
             name='is_active_user',
-            field=models.BooleanField(default=False),
+            field=models.NullBooleanField(default=False, null=True),
```

Comment:
I think we need `blank=True` here

Reply:
https://docs.djangoproject.com/en/1.11/_modules/django/db/models/fields/#NullBooleanField
Thanks @pkdash looks like null and blank are automatically set for a `NullBooleanField` so I'm actually going to remove the null=True kwarg...